app/src/main/java/br/com/catbag/gifreduxsample/asyncs/net/downloader/FileDownloader.java

```diff
@@ -31,9 +28,7 @@
     private static DownloadRoute sRoutes;
 
     public FileDownloader() {
-        if (sRoutes == null) {
-            sRoutes = new RoutesGenerator().createRoutes(DownloadRoute.class);
-        }
+        sRoutes = RetrofitBuilder.getInstance().createApiEndpoint(DownloadRoute.class, RiffsyRoutes.BASE_URL);
```

Comment:
Yeah, a default base url and a option to not pass this value on createApiEndpoint, a default_base_url can be company domain, or http://none.com